src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -152,7 +152,7 @@ def on_time_step(self, event: Event):
         if len(businesses_that_move_idx) > 0:
             # update the employer address and zipcode in self.businesses
             address_map, zipcode_map = self.addresses.get_new_addresses_and_zipcodes(
-                businesses_that_move_idx,
+                all_businesses.loc[businesses_that_move_idx],
```

Comment:
`get_new_addresses_and_zipcodes` says it requires:
`those_that_move: a pd.Series of ids (e.g. employer_id or household_id) who will move`

^ aren't you passing in a DataFrame there, with columns "age" and "household_id"?

so i think(?) it should instead be `all_businesses.loc[businesses_that_move_idx,'household_id']`?